src/extras/helpers/VertexNormalsHelper.js

```diff
@@ -61,7 +61,7 @@ THREE.VertexNormalsHelper.prototype.update = ( function () {
 
 		var keys = [ 'a', 'b', 'c' ];
 
-		this.object.updateMatrixWorld( true );
+		this.object.updateMatrixWorld();
```

Comment:
Is your intention to remove all calls to `updateMatrixWorld()`, then -- with the exception of the renderer, perhaps?